extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -290,14 +289,31 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     const pfx = enclosingContainers.includes(
         ValidAdContainerTypes['AMP-FX-FLYING-CARPET']) ||
         enclosingContainers.includes(ValidAdContainerTypes['AMP-STICKY-AD']);
+    let npa = null;
+    let consent = null;
+    switch (consentState) {
+      case undefined:
+      case CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED:
+        break;
+      case CONSENT_POLICY_STATE.INSUFFICIENT:
+        consent = false;
```

Comment:
I realize it's a line of duplication, but I think it is better readability to make each case explicit, and include npa='1' and a break in here as well.